### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by milios**

### CODE REVIEW

File: api-v7/dbag/energy/m7t/orderbook/v7/orderbooks.proto [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ * OrderBook Snapshots and Deltas sent by outbound gateway V7.
+ */
+syntax = "proto3";
+
+package dbag.energy.m7t.orderbook.v7;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "com.deutscheboerse.energy.m7.api.orderbook.v7";
```
Comment (auTasso) [inline api-v7/dbag/energy/m7t/orderbook/v7/orderbooks.proto:11]: :yellow_circle: Do we want this java_package here ?

Comment (milios) [reply api-v7/dbag/energy/m7t/orderbook/v7/orderbooks.proto:11]: welll it should be exactly the same as public api.
so we either change it in public api spec or we keep it here....

Comment (auTasso) [reply api-v7/dbag/energy/m7t/orderbook/v7/orderbooks.proto:11]: :thinking: should it ? isn't this only java package, and it matters only when locally mapping (aka java_package is not per se part of the protobuf, you can have the same protobuf type marshalled with "java_package_1" and unmarshalled with "java_package_2" no ? 
I could be wrong.

Comment (milios) [reply api-v7/dbag/energy/m7t/orderbook/v7/orderbooks.proto:11]: as agreed, we keep it

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
-# m7-orderbooks-v7-client
-Example client for the M7 Trading Orderbook v7 Api - Websockets
+# M7 Orderbook Api (v7) Client
+Example client for the M7 Orderbook (v7) Api - using Websockets.
+
+## How to Build?
+All you need is java 17 or newer :
```
Comment (auTasso) [inline README.md:5]: Java 21, no ?

File: client/src/main/resources/m7OrderbookClient.properties [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# url to the M7 Orderbook Websocket Api:
+url=ws://localhost:8080/broadcast/
+# user to connect with:
+userId=M7USR001
+# subscription details:
+# token received in m7 amqp api (v6) within the UserRprt message:
+subscription.token=OiICe1EJScQCpF4QRZ4BDT+tsYvR1AQdUAXnigOT9wMMvSaUXsgG7xdnev46rY6wMuxmrgyG8AFuf4p5umfisg
+# assignments to subscribe to (expects format like "product1:area1,product2:area2")
+subscription.assignments=60_Min_Power:10YDE-RWENET---I,60_Min_Power:10YDE-EON------1
+s
```
Comment (auTasso) [inline client/src/main/resources/m7OrderbookClient.properties:10]: Typo, floating character.

File: README.md (hunk 2 above)
Comment (milios) [reply README.md:5]: I think that would be unnecessary restriction for the users, wouldn't it ?

File: client/src/main/resources/m7OrderbookClient.properties (hunk 3 above)
Comment (milios) [reply client/src/main/resources/m7OrderbookClient.properties:10]: :+1:

**Review by auTasso — APPROVED** [review]

**Pull request merged**